ext/mbstring/ucgendat/ucgendat.php

```diff
@@ -372,6 +378,43 @@ function parseDerivedCoreProperties(UnicodeData $data, string $input) : void {
     }
 }
 
+function parseEastAsianWidth(string $input) : array {
+  $wideRanges = [];
+
+  foreach (parseDataFile($input) as $fields) {
+    if ($fields[1] == 'W' || $fields[1] == 'F') {
+      if ($dotsPos = strpos($fields[0], '..')) {
+        $startCode = intval(substr($fields[0], 0, $dotsPos), 16);
+        $endCode = intval(substr($fields[0], $dotsPos + 2), 16);
+
+        if (!empty($wideRanges)) {
+          $lastRange = $wideRanges[count($wideRanges) - 1];
+          if ($startCode == $lastRange->end + 1) {
+            $lastRange->end += ($endCode - $startCode + 1);
+            continue;
+          }
+        }
+
+        $wideRanges[] = new Range($startCode, $endCode);
+      } else {
+        $code = intval($fields[0], 16);
+
+        if (!empty($wideRanges)) {
+          $lastRange = $wideRanges[count($wideRanges) - 1];
+          if ($code == $lastRange->end + 1) {
+            $lastRange->end++;
+            continue;
+          }
+        }
+
+        $wideRanges[] = new Range($code, $code);
+      }
+    }
+  }
+
+  return $wideRanges;
+}
```

Comment:
This file seems to use 4-space indentation everywhere else.

Reply:
You're right. Don't know how I missed that. I have corrected the indentation for `generateEastAsianWidthData` as well.